controllers/user/forgotten-password.js

```diff
@@ -44,37 +38,51 @@ async function processResetRequest(req, res, user) {
     });
 }
 
-async function handleRequestReset(req, res) {
-    const errors = validationResult(req);
-    if (errors.isEmpty()) {
-        res.locals.alertMessage =
-            'Password reset requested. If the email address entered is correct you will receive instructions via email.';
-
-        try {
-            const email = purifyUserInput(req.body.username);
-            const user = await userService.findByUsername(email);
-
-            if (user) {
-                await processResetRequest(req, res, user);
-            }
-
-            renderRequestReset(req, res);
-        } catch (error) {
-            Raven.captureException(error);
-            renderRequestReset(req, res);
-        }
-    } else {
-        // Failed validation
-        res.locals.errors = errors.array();
-        res.locals.formValues = req.body;
-        return renderRequestReset(req, res);
-    }
+function renderForm(req, res, data = null, errors = []) {
+    res.render(path.resolve(__dirname, './views/forgotten-password'), {
+        formValues: data,
+        errors: errors
+    });
 }
 
 router
     .route('/')
     .all(requireUnauthed)
-    .get(renderRequestReset)
-    .post(validators.emailAddress, handleRequestReset);
+    .get(renderForm)
+    .post(async function(req, res) {
+        const validationResult = Joi.object({
```

Comment:
Validation here is only to check that the username is a valid format (e.g. an email)